src/Core/Checkout/Order/Event/OrderStateChangeCriteriaEvent.php

```diff
@@ -24,4 +26,9 @@ public function getCriteria(): Criteria
     {
         return $this->criteria;
     }
+
+    public function getContext(): Context
```

Comment:
can you add implements ShopwareEvent

https://github.com/shopware/shopware/blob/9ab99bf35c9b9f92eb4d7389cb882b82f602f50b/src/Core/Framework/Event/ShopwareEvent.php#L11

Reply:
Yes hereby added :)